rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -77,6 +77,9 @@ def required_components(cls) -> List[Type[Component]]:
         # will be converted to lowercase if lowercase is True
         "OOV_token": None,  # string or None
         "OOV_words": [],  # string or list of strings
+        # indicates whether the featurizer should use the lemma of a word for
+        # counting (if available) or not
+        "use_lemma": True,
```

Comment:
Optional: I see that it's not really common in this class, but since `use_lemma` is used more than once, I'd extract it (and other config options) to a constant.